includes/class-sensei-lesson.php

```diff
@@ -280,6 +280,21 @@ public function meta_box_setup() {
 		// Add Meta Box for Lesson Preview
 		add_meta_box( 'lesson-preview', esc_html__( 'Preview', 'sensei-lms' ), array( $this, 'lesson_preview_meta_box_content' ), $this->token, 'side', 'low' );
 
+		/**
+		 * Filters the content drip metabox toggle.
+		 *
+		 * @hook  sensei_lesson_content_drip_hide
+		 * @since 4.1.0
+		 *
+		 * @param  {bool} $hide_content_drip
+		 * @return {bool}
+		 */
+		$hide_content_drip = apply_filters( 'sensei_lesson_content_drip_hide', false );
+		if ( ! $hide_content_drip ) {
```

Comment:
We'd like to avoid sniffing for whether or not a plugin is installed. It's the main reason why we added the filter in the first place. Evergreen will need to update Content Drip and turn this off from the extension.